observability/cockpit/api-cli/configuring-grafana-agent.mdx

```diff
@@ -0,0 +1,178 @@
+5. Copy the following yaml template.
+    <Message type="important">
+     - Make sure you replace the `remote_write` URL with `https://metrics.prd.obs.fr-par.scw.cloud/api/v1/push` and the `clients` URL with `https://logs.prd.obs.fr-par.scw.cloud/loki/api/v1/push`
+     - Make sure you add the token you created in step 2
+    </Message>
+
+    ```
+    metrics:
+    wal_directory: /tmp/agent
+    global:
+        scrape_interval: 15s
+        remote_write:
+        - url: <your metrics push url>
+        headers:
+            "X-Token": <your push token>
+
+    logs:
+    positions_directory: /tmp/positions/
+    configs:
+    - name: scw
+        clients:
+        - url: <your logs push url>
+        tenant_id: <your push token>
+        scrape_configs:
+        - job_name: varlogs
+        static_configs:
+        - targets: [localhost]
+            labels:
+            job: varlogs
+            __path__: /host/root/var/log/*.log
+
+    integrations:
+    node_exporter:
+        enabled: true
+        rootfs_path: /host/root
+        sysfs_path: /host/sys
+        procfs_path: /host/proc
+    ```
+    Your configuration file should look like the following:
+
+    ```
+    metrics:
+    wal_directory: /tmp/agent
+    global:
+        scrape_interval: 15s
+        remote_write:
+        - url: https://metrics.prd.obs.fr-par.scw.cloud/api/v1/push
+        headers:
+            "X-Token": 4zz-3cGLS5tzuDRb6LkEdpem7eOqj8EFecfWcJmTvrakS15gu07iEvtC5RfYDzi8
+
+
+    logs:
+    positions_directory: /tmp/positions/
+    configs:
+    - name: scw
+        clients:
+        - url: https://logs.prd.obs.fr-par.scw.cloud/loki/api/v1/push
+        tenant_id: 4zz-3cGLS5tzuDRb6LkEdpem7eOqj8EFecfWcJmTvrakS15gu07iEvtC5RfYDzi8
+        scrape_configs:
+        - job_name: varlogs
+        static_configs:
+        - targets: [localhost]
+            labels:
+            job: varlogs
+            __path__: /host/root/var/log/*.log
+
+
+    integrations:
+    node_exporter:
+        enabled: true
+        rootfs_path: /host/root
+        sysfs_path: /host/sys
+        procfs_path: /host/proc
+    ```
+6. In the same folder, create another file and name it `docker-compose.yaml`.
+7. Copy the following template and paste it into your file:
+
+    ```
+    version: "3"
+    services:
+    agent:
+        image: grafana/agent:latest
+        restart: always
+        volumes:
+        - "./config.yaml:/etc/agent-config/agent.yaml:ro,rslave"
+        - "/:/host/root:ro,rslave"
+        - "/sys:/host/sys:ro,rslave"
+        - "/proc:/host/proc:ro,rslave"
+        - "/tmp/agent:/etc/agent"
+        entrypoint:
+        - /bin/agent
+        - -config.file=/etc/agent-config/agent.yaml
+        - -metrics.wal-directory=/tmp/agent/wal
+        network_mode: "host"
+        pid: "host"
+    ```
+    <Message type="important">
+     If you are using OSX, remove all occurrences of `,rslave` in your configuration file.
```

Comment:
Is `,rslave` mandatory under non OSX systems? if not we could just omit it and this message